net/dawn/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# This software is licensed under the Public Domain.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dawn
+PKG_VERSION:=2020-03-23
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/berlin-open-wireless-lab/DAWN.git
+PKG_SOURCE_VERSION:=f03f55ff92153337972cb9b2d1856b9a364df073
+PKG_MIRROR_HASH:=aaa21efafac48749c0cf233bc723bb42a1d1591761d33cc1cee5b27bed8adee1
+
+PKG_MAINTAINER:=Nick Hainke <vincent@systemli.org>
+PKG_LICENSE:=GPL-2.0
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_PARALLEL:=1
+CMAKE_INSTALL:=1
```

Comment:
This is actually useless. cmake.mk defines PKG_INSTALL:=1. CMAKE_INSTALL is PKG_INSTALL + InstallDev, which is not needed here.